Todo TN1.md

```diff
@@ -0,0 +1,48 @@
+Todo TN1
+---------
+
+#Obscuro:
+
+- evm /statedb - finalise & test properly (make sure it works through reorgs)
+
+- k/v on top of edgelessdb
+
+- add support for multiple supported erc20s (mapping L1/L2 address + withdrawals)
```

Comment:
I thought this was done, is there stuff missing

Reply:
we need the mapping from L1 erc20 address to L2 erc20 address and we need to add it to the withdrawal instruction